.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,59 @@
+#Copyright The Ratify Authors.
+#Licensed under the Apache License, Version 2.0 (the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+#
+#http://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+
+#template created based on verbiage from https://github.com/oras-project/oras/blob/cc40e430ee0cc124250f9855a2e99f535f759221/.github/ISSUE_TEMPLATE/bug-report.yaml
+name: Bug Report
+description: File a bug report
+labels: [bug]
+body:
+  - type: markdown
+    id: preface
+    attributes:
+      value: |
+        Thank you for reporting bugs to Ratify!
+  - type: textarea
+    id: environment
+    validations:
+      required: true
+    attributes:
+      label: What happened in your environment? 
+      description: "install ratify with debug logs by adding --set logLevel=debug to helm install command"
+  - type: textarea
+    id: expect
+    attributes:
+      label: "What did you expect to happen?"
+  - type: input
+    id: version
+    attributes:
+      label: "What version of Kubernetes are you running?"  
+  - type: input
+    id: version
+    attributes:
+      label: "What version Ratify are you running?"         
+  - type: textarea
+    id: solution
+    attributes:
+      label: "Anything else you would like to add?"
+      description: "Miscellaneous information that will assist in solving the issue."
+    validations:
+      required: true
```

Comment:
does this mean someone has to fill this field out?

Reply:
fixed thanks!